CRM/Queue/Runner.php

```diff
@@ -172,28 +172,51 @@ public function runAllInteractive(): void {
     UserJob::save(FALSE)->setRecords([$userJob])->execute();
 
     if (Civi::settings()->get('enableBackgroundQueue')) {
-      $this->runAllViaBackground();
-      return;
+      $url = $this->startBackgroundRun();
```

Comment:
@colemanw what is the reason for these changes?

Reply:
This breaks a couple larger function into smaller functions, with the effect of allowing `runAllInteractive()` to be called via ajax (in which case we do *not* want it issuing an http redirect).